### PR TITLE
Improve repository appearence

### PR DESCRIPTION
**Pull request by NoComment1105**
This PR adds a README to the project, as well as issue form and contact links to the Toolchain discord.

For the contributing file, I request that people provide general categories for pull requests and FCP durations either within this thread (preferable) or on Discord.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Quilt Kotlin Libraries
+
+#### Note: At the moment, [Quilt Standard Libraries](https://github.com/QuiltMC/quilt-standard-libraries), and as such Quilt Kotlin Libraries, are not read for general use. Please make an issue or talk to the QKL team on discord before writing any PRs.
```
Comment (Oliver-makes-code) [inline README.md:3]: This is a bit confusing to read

Comment (Jamalam360) [reply README.md:3]: Also `read` should probably be `ready`

Comment (NoComment1105) [reply README.md:3]: I'm aware :>

Comment (wafflecoffee) [reply README.md:3]: Since when was QSL not ready for general use?

Comment (NoComment1105) [reply README.md:3]: The QSL repo says that itself, so since when? when ever that was written 😛

**Review by sschr15 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+## Guide: Pull Requests
+
+1. ### Discuss your feature
+   Be sure to talk to us (either through an issue or on [Discord](https://discord.quiltmc.org/toolchain)) before working on your feature! We can help you with any questions you may have, and save some time during the review process.
+2. ### Open your PR and wait for reviews
+   Once you have forked QSL and opened a pull request, you need to wait for people to review it. When you get reviews, try to thoughtfully address any concerns other people have. If you get confused, be sure to ask questions!
+3. ### Entering a Final Comment Period
+   Once your PR has no "changes requested" reviews, the minimum number of approvals for its [triage category](#guide-triage-categories), and nobody has their review requested, it is eligible to enter a Final Comment Period (FCP). A FCP is a last call for any reviewers to look at your PR before it is merged. The minimum length of your PR's FCP is determined by its triage category, but if any further changes are requested, the FCP might be lengthened, or if the concerns are significant, the FCP cancelled until the concerns are resolved.
```
[the remark this answers is not in the document]
Comment (NoComment1105) [reply CONTRIBUTING.md:11]: That's because I asked for suggestions on such things in the PR title 🙃

**Comment by Kroppeb** [comment]
Just as an FYI: [0039-pr-policy.md](https://github.com/QuiltMC/rfcs/blob/master/structure/0039-pr-policy.md)

**Comment by SilverAndro** [comment]
Looks good other than the requested changes, any plans on finishing this out?

File: .github/ISSUE_TEMPLATES/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+        
+        Before continuing, please make sure there are no similar issues on [the issue tracker](https://github.com/QuiltMC/quilt-kotlin-libraries/issues). If there are, consider contributing your information there instead!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: 'Also tell us what did you expect to happen? PLease provide some reproduction steps: What did you do to trigger this bug?'
```
Comment (Jamalam360) [inline .github/ISSUE_TEMPLATES/bug_report.yml:16]: PLease

**Comment by triphora** [comment]
Why was I requested for review?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Contributing
+This documents outlines best-practices and contributing guidelines to the Quilt Kotlin Libraries
+
+## Guide: Pull Requests
+
+1. ### Discuss your feature
+   Be sure to talk to us (either through an issue or on [Discord](https://discord.quiltmc.org/toolchain)) before working on your feature! We can help you with any questions you may have, and save some time during the review process.
+2. ### Open your PR and wait for reviews
+   Once you have forked QSL and opened a pull request, you need to wait for people to review it. When you get reviews, try to thoughtfully address any concerns other people have. If you get confused, be sure to ask questions!
```
Comment (Jamalam360) [inline CONTRIBUTING.md:9]: QSL should probably be QKL

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+3. ### Entering a Final Comment Period
+   Once your PR has no "changes requested" reviews, the minimum number of approvals for its [triage category](#guide-triage-categories), and nobody has their review requested, it is eligible to enter a Final Comment Period (FCP). A FCP is a last call for any reviewers to look at your PR before it is merged. The minimum length of your PR's FCP is determined by its triage category, but if any further changes are requested, the FCP might be lengthened, or if the concerns are significant, the FCP cancelled until the concerns are resolved.
+4. ### Request a merge!
+   Once the minimum time on the Final Comment Period has passed, and you have resolved any concerns reviewers have raised during that time, leave a comment on your PR requesting for it to be merged. A QSL Core Team member will take a final look over your PR, and if everything looks good, merge it!
```
Comment (Jamalam360) [inline CONTRIBUTING.md:13]: Once again QSL, idk if it's intentional but it seems wrong

**Review by Jamalam360 — COMMENTED** [review]
Some typos lol, not that I was actually supposed to be requested :>

**Review by Jamalam360 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+We provide wrappers for various Minecraft features, for example Registries and Math functions. On top of this we provide
+Kotlin wrappers for [QSL](https://github.com/QuiltMC/quilt-standard-libraries) APIs, allowing better Kotlin integration
+with QSL features.
+
+
+### Repository Structure
+* The `core` folder. This contains all the core functionality for the Quilt Kotlin Libraries.
+* The `fatjar` folder. This contains the means for creating one Quilt Kotlin Libraries Jar that can be added to mods
+folders and development environments.
+* The `wrapper` folder. This contains all the Kotlin wrappers to both QSL and Minecraft features/functions.
```
Comment (sschr15) [inline README.md:20]: Maybe link the folders for them style points?

**Review by sschr15 — APPROVED** [review]

**Review by Oliver-makes-code — APPROVED** [review]

**Review by SilverAndro — APPROVED** [review]

**Review by PotatoPresident — APPROVED** [review]

**Pull request merged**